src/io/file_system.ts

```diff
@@ -110,39 +112,57 @@ export class NodeFileSystem implements tfc.io.IOHandler {
       };
     }
   }
-
   async load(): Promise<tfc.io.ModelArtifacts> {
-    if (Array.isArray(this.path)) {
-      throw new Error('Loading from multiple paths is not supported yet.');
-      // TODO(cais, nkreeger): Implement this. See
-      //   https://github.com/tensorflow/tfjs/issues/343
-    }
+    return Array.isArray(this.path) ? this.loadBinaryModel() :
+                                      this.loadJSONModel();
+  }
 
-    const info = await stat(this.path).catch(doesNotExistHandler('Path'));
+  protected async loadBinaryModel(): Promise<tfc.io.ModelArtifacts> {
+    const topology =
+        await stat(this.path[0]).catch(doesNotExistHandler('Path'));
+    const weightManifest =
+        await stat(this.path[1]).catch(doesNotExistHandler('Path'));
 
     // `this.path` can be either a directory or a file. If it is a file, assume
     // it is model.json file.
+    if (topology.isFile() && weightManifest.isFile()) {
```

Comment:
Similar theme here, might be easier for the user to debug problems if the code looked like this:

```ts
if (!topology.isFile()) {
  throw new Error('File specified for topology is not a file!');
}
if (!weightManifest.isFile()) {
  throw new Error('File specified for the weight manifest is not a file!');
}
```